src/css/application.css

```diff
@@ -420,13 +421,13 @@ body {
 }
 
 .editors__column-divider {
-  background-color: var(--color-dark-gray);
+  background-color: #aaa;
```

Comment:
@ibrand would it make sense to label the grays that _don’t_ provide sufficient contrast, to make clear that they should only be used for accent/aesthetics?

E.g.:

```css
  --color-low-contrast-gray: #eee;
  --color-light-gray: #aaa;
  --color-gray: #666;
  --color-dark-gray: #444;
```